### DISCUSSION TITLE
Maximum par diff not valid

### DISCUSSION THREAD

**Discussion by wkitlasten** [usgs/pestpp#258] [General]
Hey, 

I am playing with a sequential approach to HM. I fixed some par groups and I am now getting the following ('22' realz reported before crashing). 

`raised an exception: multimodal solve error: maximum par diff for realization '0' not valid`

I am using 300 realz, mm_alpha=0.2, and have 22,496 adjustable parameters. I did not get this exception with 120,000 pars. Wondering if this is expected with fewer pars or if I (probably) messed something else up. 

Not sure if I can change "max par diff" (and I doubt I want to), or if there is a better way to address this issue?

**Comment by jtwhite79** [comment]
Thats pretty suss - I dont see how that be raised without something buggy happening. Basically it means that the maximum euclidean distance between real '0' and all other reals came up less than or equal to 0.0:

[https://github.com/usgs/pestpp/blob/a3137fe0177821c1ded8c20d9a680ad9ca5408cb/src/libs/pestpp_common/EnsembleMethodUtils.cpp#L182](https://github.com/usgs/pestpp/blob/a3137fe0177821c1ded8c20d9a680ad9ca5408cb/src/libs/pestpp_common/EnsembleMethodUtils.cpp#L182)

Can you send me your master dir files so I can have a look?
